Add guide for sink to S3 with RoleARN (#22488)
pingcap/docs: tidb-cloud/changefeed-sink-to-cloud-storage.md

## Step 1. Configure destination

> Navigate to the cluster overview page of the target TiDB cluster. Click **Data** > **Changefeed** in the left navigation pane, click **Create Changefeed** to go to the **Configure Destination** page, and then select **Amazon S3**, **GCS**, or **Azure Blob Storage** as the destination. The configuration process varies depending on the destination you choose.

Reviewer: ```suggestion
Navigate to the cluster overview page of the target TiDB cluster. Click **Data** > **Changefeed** in the left navigation pane, click **Create Changefeed** to go to the **Destination** page, and then select **Amazon S3**, **GCS**, or **Azure Blob Storage** as the destination, depending on the cloud provider on which your cluster is hosted. The configuration process varies depending on the destination you choose.
```

<SimpleTab>
<div label="Amazon S3">

For **Amazon S3**, you can use either **AWS Role ARN** or **AWS access key** for authentication. Using **AWS Role ARN** is recommended for stronger security and easier management.

**Option 1: AWS Role ARN (recommended)**

To use an IAM Role for authentication, follow these steps:

> 1. On the **Configure Destination** page for Amazon S3, enter the **S3 URI**. Make sure that the S3 bucket is in the same AWS region as your TiDB cluster.

Reviewer: ```suggestion
1. On the **Destination** page for Amazon S3, enter the **S3 URI**. Make sure that the S3 bucket is in the same AWS region as your TiDB cluster.
```

3. To create a new Role ARN, click **Click here to create new one with AWS CloudFormation**. This template automatically configures the required permissions.

    If you prefer to create the role manually, click **Create Role ARN manually** to view the TiDB Cloud account information and the required policy.

4. Ensure your IAM role has at least the following permissions for the target bucket:

    - `s3:ListBucket`
    - `s3:PutObject`
    - `s3:GetObject`
    - `s3:DeleteObject`

5. Paste the generated **Role ARN** into the corresponding field.

**Option 2: AWS access key**

> **Note:**
>
> Using an access key and secret key (AK/SK) requires manual credential management and rotation, which increases security risks. For stronger security, it is recommended to use **AWS Role ARN** instead.

To use an access key for authentication, follow these steps:

1. On the **Configure Destination** page for Amazon S3, enter the **S3 URI**. Make sure that the S3 bucket is in the same AWS region as your TiDB cluster.
2. Under **Bucket Access**, select **AWS Access Key**.
3. Fill in the following fields:

    - **Access Key ID**
    - **Secret Access Key**

> 

Reviewer: Change **Configure Destination** page to **Destination** page on [L122](https://github.com/pingcap/docs/pull/22488/files#diff-03bff1ba3daf71c27b65ff02be616e4e401316a5f68adfcd805935e7a12b78bdR154)

<SimpleTab>
<div label="Amazon S3">

For **Amazon S3**, you can use either **AWS Role ARN** or **AWS Access Key** for authentication. Using **AWS Role ARN** is recommended for better security and easier management.

Make sure that the S3 bucket is in the same region as your TiDB cluster.

> #### Option 1: AWS Role ARN (Recommended)

Reviewer 2: ```suggestion
**Option 1: AWS Role ARN (Recommended)**
```

To use an IAM Role for authentication, follow these steps:

1.  In the TiDB Cloud console, go to the **Configure Destination** page for the S3 sink.

> 2.  Fill in the **S3 URI**.

Reviewer 2: ```suggestion
2.  Enter the **S3 URI**.
```

3.  Under **Bucket Access**, select **AWS Role ARN**.
4.  To create a new Role ARN, click **Click here** to use the AWS CloudFormation template. This will automatically configure the necessary permissions.
    - If you prefer to create the role manually, click **Create Role ARN manually** to view the TiDB Cloud account information and the required policy.
5.  Ensure your IAM role has at least the following permissions for the target bucket:
    - `s3:ListBucket`
    - `s3:PutObject`
    - `s3:GetObject`
    - `s3:DeleteObject`
6.  Paste the generated **Role ARN** into the corresponding field.

![s3_endpoint](/media/tidb-cloud/changefeed/sink-to-cloud-storage-s3-endpoint-arn.jpg)

#### Option 2: AWS Access Key

> [!IMPORTANT]
> Using an access key and secret key (AK/SK) requires manual credential management and rotation, which increases security risks. For better security, use an **AWS Role ARN** instead.

> To use an access key, fill in the following fields:

Reviewer 2: ```suggestion
To use an access key, fill in the following fields:

```

<SimpleTab>
<div label="Amazon S3">

For **Amazon S3**, you can use either **AWS Role ARN** or **AWS access key** for authentication. Using **AWS Role ARN** is recommended for stronger security and easier management.

Make sure that the S3 bucket is in the same AWS region as your TiDB cluster.

**Option 1: AWS Role ARN (recommended)**

To use an IAM Role for authentication, follow these steps:

1.  In the TiDB Cloud console, go to the **Configure Destination** page for the S3 sink.
2.  Enter the **S3 URI**.
3.  Under **Bucket Access**, select **AWS Role ARN**.

> 4.  To create a new Role ARN, click **Click here** to use the AWS CloudFormation template. This template automatically configures the required permissions.

Reviewer 2: ```suggestion
4.  To create a new Role ARN, click **Click here to create new one with AWS CloudFormation**. This template automatically configures the required permissions.
```
